speech/check_dataVals.m

```diff
@@ -414,51 +487,48 @@ function update_plots(src,evt)
 function pick_line(src,evt,iLine,iPlot)
     UserData = guidata(src);
     unselectedColor = [0.7 0.7 0.7];
-    f1color = [0 0 1]; % blue
-    f2color = [1 0 0]; % red
+    plotcolor{1} = [0 0 1]; % blue
+    plotcolor{2} = [1 0 0]; % red
```

Comment:
Is this the only thing restricting plotting more than 2 signals? Could add more colors if so

Reply:
Added 3rd plot color in c2f23bf3d389ad07c2f5576d28dc3eb2257793da